common/shlibs

```diff
@@ -16,6 +16,7 @@
 # PLEASE NOTE: when multiple packages provide the same SONAME, the first
 # one (order top->bottom) is preferred over the next ones.
 #
+libKF5KDEGames.so.7 libkdegames-18.08.0_1
```

Comment:
please add it to the bottom of the file and only if there are packages that require it